ContentPatcher/docs/i18n/zh/author-guide/action-editmap.md

```diff
@@ -0,0 +1,383 @@
+一个`EditImage`补丁是`Changes`下含有此字段的模型 (见例子)。所有更改类型都需要这些字段。
+
+<dl>
+<dt>必填字段：</dt>
+<dd>
+
+字段       | 用途
+--------- | -------
+`Action`  | 要进行的更改类型。此操作类型设置为`EditMap`。
+`Target`  | 需编辑的[游戏素材名](../author-guide.md#what-is-an-asset)（或多个由逗号分隔的素材名），比如`Maps/Town`。该字段支持[令牌](../author-guide.md#tokens)，不区分大小写。
+
+</dd>
+<dt>可选字段：</dt>
+<dd>
+
+字段       | 用途
+--------- | -------
+`When`      | _(可选)_ 使此补丁只有在指定[条件](../author-guide.md#conditions)下生效.
+`LogName`   | _(可选)_ 此补丁在日志里显示的名字，有助于理解报错。默认为类似`EditImage Maps/Town`的名字。
+`Update`    | _(可选)_ 此补丁字条的更新频率，详见[update rate](../author-guide.md#update-rate)。
+`LocalTokens` | _(可选)_ 一组仅在此补丁中生效的[局部令牌](../author-guide/tokens.md#local-tokens)。
+
+
+</dd>
+<dt>进阶字段：</dt>
+<dd>
+
+<table>
+  <tr>
+    <td>字段</td>
+    <td>用途</td>
+  </tr>
+  <tr>
+  <td><code>Priority</code></td>
+  <td>
+
+ _（可选）_ 当多个补丁编辑同一数据素材时，此字段控制它们应用的顺序。可用的值有`Early`（更早），`Default`（默认），还有`Late`（更晚）。默认值为`Default`。
+
+补丁（包括所有模组）按以下顺序生效：
+
+1. 优先级从早到晚；
+2. 按照模组加载顺序（基于依赖关系）；
+3. 按照补丁在`content.json`中列出的顺序。
+
+如果需要更具体的顺序，可以使用简单的偏移量，如`"Default + 2"`或者`"Late - 10"`。
+默认值为-1000 （`Early`），0（`Default`）和1000（`Late`）。
+
+此字段 _不_ 支持令牌，不区分大小写。
+
+> [!TIP]
+> 优先级会让你的更改难以排除故障。推荐做法：
+> * 如果可以的话，只使用上述无偏移的优先级（比如外观覆盖设为`Late`）
+> * 在 _你自己_ 的补丁里不需要用优先级，因为你可以自己在content.json排列好补丁应用的顺序。
+
+  </tr>
+  <tr>
+  <td><code>TargetLocale</code></td>
+  <td>
+
+ _（可选）_ 素材名称中要匹配的地区代码，比如设置`"TargetLocale": "fr-FR"`只编辑法语形式的素材（比如`Data/Achievements.fr-FR`）。可以为空，只有只编辑没有地域区分的基本素材。
+
+如果省略，它将应用于所有素材，不管有没有本地化。
+
+</td>
+</table>
+</dd>
+</dl>
+
+可选以下某一或多个段落的字段。
+
+### 地图叠加<a name="overlay-a-map"></a>
+
+一个‘地图叠加'型更改将图块，属性，和图块表从源地图拷贝到目标地图。目标区域下对应的图层将被源地图完全覆盖。
+
+此补丁的字段为：
+
+<table>
+<tr>
+<th>字段</th>
+<th>用途</th>
+</tr>
+<tr>
+<td>&nbsp;</td>
+<td>
+
+详见以上的 _[公共字段](#common-fields)_
+
+</td>
+</tr>
+<tr>
+<td>
+
+`FromFile`
+
+</td>
+<td>
+
+内容包文件夹中要修补到目标中的图像的相对路径（例如`assets/town.tmx`），或多个逗号分隔的路径。这可以是`.tbin`，`.tmx`，或`.xnb`文件。该字段支持[令牌](../author-guide.md#tokens)，不区分大小写。
+
+Content Patcher会如下处理`FromFile`地图内引用的图块表：
+* 若图块表没有被目标地图引用，Content Patcher会帮你添加此图块（并自动添加`z_` ID 前缀，以避免与硬编码的游戏逻辑冲突）。如果源地图具有已引用的图块表的自定义版本，则它将被添加为仅供你的图块使用的单独图块表。
+* 如果你模组文件夹里包含你的图块表，Content Patcher会自动使用它；否则它将从游戏的`Content/Maps`文件夹中加载。
+
+</td>
+</tr>
+<tr>
+<td>
+
+`FromArea`
+
+</td>
+<td>
+
+_（可选）_ 源地图中需拷贝到目标的部分，默认整个源地图
+
+此字段是一个含有左上角点的X和Y像素坐标区域的长（`Width`）与高（`Height`）的对象。该对象的字段支持[令牌](../author-guide.md#tokens)。
+
+</td>
+</tr>
+<tr>
+<td>
+
+`ToArea`
+
+</td>
+<td>
+
+_（可选）_ 目标地图中要替换的部分。默认大小与 `FromArea` 相同，位于地图的左上角。
+
+此字段是一个含有左上角点的X和Y像素坐标区域的长（Width）与高（Height）的对象。该对象的字段支持[令牌](../author-guide.md#tokens)。
+
+果你指定的区域超出了地图的底部或右部，Content Patcher将自动调整地图大小以适应新地图。
+
+</td>
+</tr>
+<tr>
+<td>
+
+`PatchMode`
+
+</td>
+<td>
+
+_（可选）_ 何将 `FromArea` 应用于 `ToArea`。默认为 `ReplaceByLayer`。
+
+例如，假设你有一个大部分为空的源地图，包含两个图层：`Back`（红）和`Buildings`（蓝）：
+
+![](../../../screenshots/map-patch-mode-source.png)
+
+以下是它们在不同`PatchMode`下的组合（黑色区域代表地图背后的虚空，游戏内显示为黑）：
+
+* **`Overlay`**  
+  只替换对应的图块。`Back`图层的红图块代替了`Back`图层的地面，而`Buildings`图层的蓝图块没有替换任何图块，地面依旧可见。
```

Comment:
The description for the `Overlay` patch mode might be incorrect:
> **en:** The red tile replaces the ground on the `Back` layer, but the ground is visible under the blue `Buildings` tile.
> **zh:** 只替换对应的图块。`Back`图层的红图块代替了`Back`图层的地面，而`Buildings`图层的蓝图块没有替换任何图块，地面依旧可见。

The translation seems to say that the `Buildings` layer doesn't replace any tiles; if so that's incorrect, since it replaces the tile on the `Buildings` layer (just not the `Back` ground tile under it).